docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -0,0 +1,247 @@
+```
+
+![npx_prisma_migrate](https://storage.googleapis.com/bigcommerce-production-dev-center/images/npx_prisma_migrate.png)
+
+3. Start the app server.
+
+```shell title="Start the app server"
+npm run dev
+```
+
+4. Start ngrok using the follow command. If your app does not run on port 3000, replace `3000` with your app server's port.
+
+```shell title="Start ngrok"
+ngrok http 3000
+```
+
+Once the app server and ngrok are running, you can install the draft app on your sandbox store. For more about installing and troubleshooting apps in development, see our App Tutorial section on [Installing and launching an app](/api-docs/apps/tutorials/sample-app-nextjs/step-2-connect#install-and-launch-the-app).
+
+## Replacing SQLite
+
+To use an alternate SQL database, do the following:
+
+1. Update the `/prisma/schema.prisma` file with a `provider` other than `SQLite`. Consult [Prisma's reference docs](https://prisma.io/docs/reference/tools-and-interfaces/prisma-schema/data-sources/) for a list of options.
+
+2. In `/prisma/.env`, update the value of the `DATABASE_URL` variable to match your new database connection string.
+
+3. Run `npx prisma migrate dev` per the preceding section.
+
+4. Run `npx prisma generate` to generate a fresh app client that uses the new database provider.
+
+5. To access this database locally, run `npx prisma studio` and use a visual editor to verify that the tables have been created.
+
+6. Start the app server and then ngrok, per the preceding section.
+
+
+## Managing subscription products
+
+To add new subscription rules and edit existing ones, go to the **Channel Manager** menu in the store control panel and click the **Stripe Subscriptions** channel. Click on the product you want to modify. For more information, see our support article on [Foundations for Stripe Billing](https://support.bigcommerce.com/s/article/Connecting-Stripe-Payment-Gateway?language=en_US#billing).
+
+<!-- theme: info -->
+> #### Subscription sales channel
+> The app will create a dedicated sales channel for itself upon installation.
+
+If you plan to use the API to add products to the subscription sales channel, learn more about [product channel assignments](/api-docs/multi-storefront/api-guide#products).
+
+## Troubleshooting
+
+### Environment variable not found when running 
+
+This error can occur when the `/prisma/schema.prisma` file contains the incorrect provider, or the `.env` file contains an incorrect `DATABASE_URL`.
+
+### A Not found error during app installation
+
+If you don't enable the proper OAuth scopes, the `/api/auth` request might fail. Go to the [Dev Portal](https://devtools.bigcommerce.com) to check out the OAuth scopes enabled in the app's profile; consult our article on [Managing Apps in the Developer Portal](/api-docs/apps/guide/developer-portal#edit-technical-details) to learn more. Make sure your scopes equal to or exceed those listed in the preceding section on [Creating an app profile](#create-an-app-profile).
```

Comment:
I would say something like: "Go to the ... to verify that the proper OAuth scopes are enabled in the profile of your app. To learn more, read "Managing Apps..... "

Remove "preceding section" and instead say "exceed those listed in the [Creating an app profile] section."